features/step_definitions/sample_file_steps.rb

```diff
@@ -135,7 +135,7 @@ def initialize
 end
 
 Given(/^a masking configuration file in the HOME directory$/) do
-  set_env('HOME', File.expand_path(File.join(current_directory, 'home')))
+  set_env 'HOME', Pathname("#{__dir__}/../../#{current_directory}/home").to_s
```

Comment:
Argh, good catch, apologies – in all the other places switching to `__dir__` meant the code will work regardless of whether it was ran in reek’s checkout dir or not and I got a bit too over-zealous. Will fix!